generate/lang/scrambled.py

```diff
@@ -26,6 +26,8 @@ def scramble(key: str, value: str, cache: dict[str, str]) -> str:
     if key == "language":
         return "Scrambled"
     
+    random.seed = key
```

Comment:
This is how you set the seed for the random
```suggestion
    random.seed(key)
```